dashboard/app/controllers/plc/courses_controller.rb

```diff
@@ -0,0 +1,75 @@
+class Plc::CoursesController < ApplicationController
+  load_and_authorize_resource except: [:create]
+  before_action :set_plc_course, only: [:show, :edit, :update, :destroy]
+
+  # GET /plc/courses
+  # GET /plc/courses.json
+  def index
+    @plc_courses = Plc::Course.all
+  end
+
+  # GET /plc/courses/1
+  # GET /plc/courses/1.json
+  def show
+  end
+
+  # GET /plc/courses/new
+  def new
+    @plc_course = Plc::Course.new
+  end
+
+  # GET /plc/courses/1/edit
+  def edit
+  end
+
+  # POST /plc/courses
+  # POST /plc/courses.json
+  def create
+    @plc_course = Plc::Course.new(plc_course_params)
+
+    respond_to do |format|
+      if @plc_course.save
+        format.html { redirect_to @plc_course, notice: 'Course was successfully created.' }
+        format.json { render action: 'show', status: :created, location: @plc_course }
+      else
+        format.html { render action: 'new' }
+        format.json { render json: @plc_course.errors, status: :unprocessable_entity }
```

Comment:
Do you need all these actions to respond to JSON?  If not you can probably remove the extra `format` boilerplate and always return HTML.